contracts/multisig/src/signing.rs

```diff
@@ -62,10 +61,9 @@ fn validate_session_signature(
     Ok(())
```

Comment:
```suggestion
    match key.pub_keys.get(&signer) {
        Some(pub_key) if !signature.verify(&session.msg, pub_key)? => {
            Err(ContractError::InvalidSignature {
                session_id: session.id,
                signer,
            })
        }
        None => Err(ContractError::NotAParticipant {
            session_id: session.id,
            signer,
        }),
        _ => Ok(()),
    }
```
Replace the above match with.